flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/rules/FlinkStreamRuleSets.scala

```diff
@@ -365,6 +365,8 @@ object FlinkStreamRuleSets {
     PythonCorrelateSplitRule.INSTANCE,
     // merge calc after calc transpose
     FlinkCalcMergeRule.INSTANCE,
+    // remove trivial calc
+    FlinkLogicalCalcRemoveRule.INSTANCE,
```

Comment:
Why not use `CoreRules.CALC_REMOVE` directly ?
please also add this rule to `FlinkBatchRuleSets`

Reply:
`Calc_REMOVE` uses the `CalcRemoveRule.Config.DEFAULT` to generate the remove rule, which captures the `LogicalCalc` rather than `FlinkLogicalCalc` (`FlinkLogicalCalc` is not a sub class of the `LogicalCalc`).

Sure. I will add it.